src/AddTodoForm.js

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
 
+const AddTodoForm = ({ onAddTodo }) => {
+    const [todoTitle, setTodoTitle] = React.useState();
```

Comment:
In order to have a controlled component you would need to initialize the state with a value. In this case it can be empty string.

That is why you're having this warning in the console:

![warning-elena](https://user-images.githubusercontent.com/70614187/232961159-4466a1b3-d459-44ba-b576-ab94d76ee18d.png)